package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "asap",
+  "version": "1.0.0",
+  "description": "HueHueHue",
```

Comment:
The project description is currently "HueHueHue", which is not very informative. Consider providing a more detailed description of the project. [medium]